Cargo.toml

```
@@ -88,6 +88,7 @@ members = [
 	"frame/election-provider-support",
 	"frame/election-provider-support/solution-type",
 	"frame/election-provider-support/solution-type/fuzzer",
+	"frame/election-provider-support/onchain",
```

```suggestion
	"frame/election-provider-support/onchain-benchmarking",
```

this crate only does benchmarking, it should be named such.